src/haproxy.py

```diff
@@ -38,8 +40,8 @@
 logger = logging.getLogger()
 
 
-class HaproxyServiceStartError(Exception):
-    """Error when starting the haproxy service."""
+class HaproxyServiceRestartError(Exception):
+    """Error when restarting the haproxy service."""
```

Comment:
Seems unused now, but might be needded and renamed to "Reload" instead of restart